### PR TITLE
make changes to some text

### PR DESCRIPTION
**Pull request by uchxchi**
- i added my name
- i changed whatever whatever

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
                 <a href="/index.html">About me</a>
             </li>
             <li>
-                <a href="/movies.html">My movies</a>
+                <a href="/movies.html">My fav movies</a>
```
Comment (kip-guile) [inline dist/index.html:18]: this is crazy. why would you do this?

File: dist/index.html [hunk 2]
```diff
@@ -42,7 +42,7 @@ <h1 class="intro-h1" id="text"><span class="Intro-name1">Nwabia Uchechi</span></
 
         <section class="bottom" >
 
-        <footer class="footer" id="text">&copy;Copyright 2022</footer>
+        <footer class="footer" id="text">&copy;Copyright 2023</footer>
```
Comment (kip-guile) [inline dist/index.html:45]: oh this is nice

File: dist/movies.html [hunk 3]
```diff
@@ -24,7 +24,7 @@
         </nav>
 
         <section>
-            <h1 class="movies-h1" id="text">My top six movies</h1>
+            <h1 class="movies-h1" id="text">My top movies</h1>
```
Comment (kip-guile) [inline dist/movies.html:27]: cool

**Review by kip-guile — COMMENTED** [review]
you try sha

File: dist/index.html (hunk 1 above)
Comment (uchxchi) [reply dist/index.html:18]: fuck you

File: dist/index.html (hunk 2 above)
Comment (uchxchi) [reply dist/index.html:45]: i dont care

**Pull request merged**